src/csvcubed/utils/sparql_handler/sparql_queries/select_data_set_dsd_csv_url.sparql

```diff
@@ -1,13 +1,15 @@
 PREFIX qb: <http://purl.org/linked-data/cube#>
 PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
 PREFIX csvw: <http://www.w3.org/ns/csvw#>
 
-SELECT ?dataSet ?dsd ?csvUrl
+SELECT ?dataSet ?dsd ?dataSetLabel ?csvUrl
```

Comment:
Do we need this label in here anymore?

Reply:
This label is here because we took it from another query (named select_dsd_dataset_label_and_uri) then deleted that query. But if we don't need the label at all, we can remove it from here as well.